server/parsec/logging.py

```diff
@@ -79,6 +79,7 @@ def info_with_debug_extra(
         def info_with_debug_extra(
             self, event: str, debug_extra: dict[str, Any], **kwargs: Any
         ) -> None:
+            assert "event" not in debug_extra
```

Comment:
Maybe rename `event` to `_event` to avoid crashing?

Reply:
:+1: